e2e/pkg/e2eclient/client.go

```diff
@@ -242,7 +242,9 @@ type GraphQLAPIRequest struct {
 
 type GraphQLResponse struct {
 	Data   json.RawMessage `json:"data"`
-	Errors []interface{}   `json:"errors"`
+	Errors []struct {
+		Message string `json:"message"`
```

Comment:
You test against the public API. Message is not part of the public API. You assert `reason`.